packages/foreman/foreman-release-scl/tfm-ror51.repo

```diff
@@ -1,10 +1,6 @@
-# As defined by https://copr.fedorainfracloud.org/coprs/g/theforeman/tfm-ror51/repo/epel-7/group_theforeman-tfm-ror51-epel-7.repo
-[group_theforeman-tfm-ror51]
-name=Copr repo for tfm-ror51 owned by @theforeman
-baseurl=https://copr-be.cloud.fedoraproject.org/results/@theforeman/tfm-ror51/epel-7-$basearch/
-type=rpm-md
-gpgcheck=1
-gpgkey=https://copr-be.cloud.fedoraproject.org/results/@theforeman/tfm-ror51/pubkey.gpg
-repo_gpgcheck=0
+[tfm-ror51]
+name=Rails 5.1 SCL
+baseurl=http://yum.theforeman.org/rails/foreman-nightly/el7/$basearch
```

Comment:
You named the directory `nightly`, not `foreman-nightly`. I like `foreman-nightly` more.

Reply:
Fair - updated